starter-code/mongoose-movies/routes/celebrities.js

```diff
@@ -0,0 +1,58 @@
+/*jshint esversion: 6 */
+
+const express = require('express');
+const Celebrity = require('../models/celebrity');
+
+const router  = express.Router();
+
+router.get('/', (req, res, next) => {
+  Celebrity.find({}, (err, celebrities) => {
+     if (err) { return next(err); }
+     else res.render('celebrities/index', {
+       celebrities: celebrities
+     });
+   });
+});
+
+router.get('/:id', (req, res, next) =>{
```

Comment:
you could render this one like this:
**router.get('/:id', function(req, res, next) {
    Celebrity.findOne({"_id" : req.params.id},(err, celebrity) => {
        if (err) {
            next(err);
        } else {
            res.render('celebrities/show', { celebrity });
        }
    });   
});**